airflow/providers/opsgenie/operators/opsgenie.py

```diff
@@ -208,3 +208,55 @@ def execute(self, context: 'Context') -> None:
             payload=self._build_opsgenie_close_alert_payload(),
             kwargs=self.close_alert_kwargs,
         )
+
+
+class OpsgenieDeleteAlertOperator(BaseOperator):
+    """
+    This operator allows you to Delete alerts to Opsgenie.
+    Accepts a connection that has an Opsgenie API key as the connection's password.
+    This operator sets the domain to conn_id.host, and if not set will default
+    to ``https://api.opsgenie.com``.
+
+    Each Opsgenie API key can be pre-configured to a team integration.
+    You can override these defaults in this operator.
+
+    .. seealso::
+        For more information on how to use this operator, take a look at the guide:
+        :ref:`howto/operator:OpsgenieDeleteAlertOperator`
+
+    :param opsgenie_conn_id: The name of the Opsgenie connection to use
+    :param identifier: Identifier of alert which could be alert id, tiny id or alert alias
+    :param identifier_type: Type of the identifier that is provided as an in-line parameter.
+        Possible values are 'id', 'alias' or 'tiny'
```

Comment:
Is this something that should be enforced in the operator itself or does the underlying API throw an error for an expected `indentifier_type` value?

Reply:
We shouldn't enforce it from Airflow side. Setting enforcement means that if the API changes the accepted values we will also have to make code changes.